eqcorrscan/utils/plotting.py

```diff
@@ -1281,7 +1281,8 @@ def noise_plot(signal, noise, normalise=False, **kwargs):
 
 
 @additional_docstring(plotting_kwargs=plotting_kwargs)
-def pretty_template_plot(template, background=False, picks=False, **kwargs):
+def pretty_template_plot(template, background=False, picks=False, event=False,
```

Comment:
I'm not keen on having both a `picks` argument and an `event` argument - how much would need to change to get rid of the `picks` argument. We can easily add a warning a la:
```python
picks = kwargs.get("picks", None)
if picks:
    Logger.warning(
        "The picks argument is depreciated, please use a full event. "
        "This argument will be removed in future versions.")
else:
    picks = event.picks
```

Reply:
This was just about being backwards compatible with the earlier invocation of pretty_template_plot. Generally, if one creates templates, then the `picks` is taken straight from `event.picks` - so it shouldn't be an issue to just supply `event` to the function.

Reply:
Yeah, good to be onto backwards compatibility! I think the way I have suggested would be compatible and start to push people towards the updated use for this function (which is better - thank you!).

Reply:
You're right - I hadn't gotten my head around all the options all the way, but now it's fixed as you suggested.